esp/public/media/scripts/program/modules/scheduling_checks.jsx

```diff
@@ -28,8 +37,29 @@ var SchedulingCheck = React.createClass({
       open: false,
       failed: false,
       timestamp: "never",
+      tableState: {
+        greyed: {},
+        sort: -1,
+        reverse: false
+      }
     };
   },
+  
+  updateTableState: function (state) {
+    this.state.tableState = state;
+  },
+  
+  resetTable: function () {
+    this.setState({
+      tableState: {
+        greyed: {},
+        sort: -1,
+        reverse: false
+      },
+      open: false
```

Comment:
Read below -- I don't think having rendering the child update the parent's state is a good strategy anyway, and I don't think it's any harder to do things right.